Mods/EpipEncounters_7d32cb52-1cfd-4526-9b84-db4867bf9356/Story/RawFiles/Lua/Epip/QuickInventory/Filters/Equipment.lua

```diff
@@ -435,6 +500,7 @@ UI.Events.RenderSettings:Subscribe(function (ev)
 
         UI.RenderSetting(QuickInventory.Settings.ItemSlot) -- Equipment slot
         UI.RenderSetting(QuickInventory.Settings.Rarity) -- Rarity
+        UI.RenderSetting(QuickInventory.Settings.Sorting) -- Sorting
```

Comment:
I'm unconvinced this needs to be a setting within the UI itself. I get the reasoning between the two sorting modes, but it's unlikely for the user to want to switch between these often. Inserting this new setting in the UI at the top makes sense (order-wise), however it pushes other, more "common-to-chance" settings down in the small viewport.

I think it would be best to put this setting in the Epip settings menu instead, where the "Close UI after using an item" and "Close upon world interaction" settings already are, so as not to bloat what are supposed to be the filter settings in the UI itself. See `EpipSettings.lua` (cringe but workable)